ClientSupport/src/main/java/io/deephaven/clientsupport/gotorow/SeekRow.java

```diff
@@ -46,37 +49,38 @@ public SeekRow(long startingRow, String columnName, Object seekValue, boolean in
     }
 
     @Override
-    @ConcurrentMethod
```

Comment:
It's right to remove this. We should hold a shared update graph lock, or do a snapshot.